src/Psalm/Internal/Cli/Psalm.php

```diff
@@ -652,29 +651,34 @@ private static function generateBaseline(
     ): array {
         fwrite(STDERR, 'Writing error baseline to file...' . PHP_EOL);
 
+        $errorBaseline = is_string($options['set-baseline']) ? $options['set-baseline'] :
```

Comment:
We're using snake_case for variable / property names in the Psalm codebase.

Reply:
Done